native-oci-generative-ai/README.md

```diff
@@ -0,0 +1,56 @@
+# OCI Native Generative AI Example
```

Comment:
Once the PR is ready for merge, please add an entry for this demo in the top-level README.